omod/src/main/java/org/openmrs/module/fhir2/narrative/OpenMRSNarrativeTemplateManifest.java

```diff
@@ -117,10 +118,6 @@ public static OpenMRSNarrativeTemplateManifest forManifestFileLocation(Collectio
 		return forManifestFileContents(manifestFileContents);
 	}
 	
-	public static OpenMRSNarrativeTemplateManifest forManifestFileContents(String... resources) throws IOException {
-		return forManifestFileContents(Arrays.asList(resources));
-	}
```

Comment:
Removed the `forManifestFileContents(String... resources)`, `forManifestFileLocation(String... propertyFilePaths)` methods since they were not used anywhere.